core/app/modules/kit-library/assets/js/components/bottom-promotion.js

```diff
@@ -0,0 +1,19 @@
+import { Text, Button } from '@elementor/app-ui';
+
+import './bottom-promotion.scss';
+
+export default function BottomPromotion() {
```

Comment:
Maybe rename it to `EnvatoPromotion`?
Unless there is no way we're gonna have another promotion at the bottom

Reply:
You right, Done.